examples/persistent-volumes/simpletest/README.md

```diff
@@ -0,0 +1,69 @@
+# How To Use Persistent Volumes
```

Comment:
This will be written better.   I just wanted to remove the placeholder stuff in here previously.